.github/workflows/data-validate.yml

```diff
@@ -21,5 +21,10 @@ jobs:
         npm install -g @actions/core@1.2.0
         npm link @hapi/joi
         npm link @actions/core
+        mv src/data.js src/tmp-data.js
```

Comment:
I would be tempted to move all the `mv` & shell/git stuff into a `scripts/data-validate.sh` so that it can be run locally as well as in the GitHub action

Contents would be:

```sh
mv src/data.js src/tmp-data.js
git fetch origin master
git restore --source=FETCH_HEAD src/data.js
mv src/data.js src/master-data.js
mv src/tmp-data.js src/data.js
node ./scripts/data-validate.js
```

Reply:
I also will add a check if is run on master will run for all users.